### ISSUE TITLE
SA1137 suggests wrong fixes

### ISSUE THREAD

**Issue by ravicini** [DotNetAnalyzers/StyleCopAnalyzers#2774]
Updated to 1.1.0-beta009 (from 1.1.0-beta008)

We now have several violations on rule SA1137 (Elements should have the same indentation)

An example is:
``` csharp
private static Uri GetTemporarySharedReadAccessUri(ICloudBlob cloudBlob)
{
    var sasToken = cloudBlob.GetSharedAccessSignature(new SharedAccessBlobPolicy {
        Permissions = SharedAccessBlobPermissions.Read,
        SharedAccessExpiryTime = DateTimeOffset.UtcNow.AddHours(1),
        SharedAccessStartTime = DateTimeOffset.UtcNow.AddMinutes(-5) // mitigate clock skew
    });

    return new Uri($"{cloudBlob.Uri}{sasToken}");
}
```

When I apply the suggested fix I get:

``` csharp
private static Uri GetTemporarySharedReadAccessUri(ICloudBlob cloudBlob)
{
    var sasToken = cloudBlob.GetSharedAccessSignature(new SharedAccessBlobPolicy {
        Permissions = SharedAccessBlobPermissions.Read,
        SharedAccessExpiryTime = DateTimeOffset.UtcNow.AddHours(1),
        SharedAccessStartTime = DateTimeOffset.UtcNow.AddMinutes(-5) // mitigate clock skew
});

    return new Uri($"{cloudBlob.Uri}{sasToken}");
}
```

**Look for the moved `});` element.**

I could provide several different examples.

**Comment by sharwell** [comment]
Looks like the new braces check is missing the test that the reference element (opening brace in this case) is the first element on the line.

**Comment by AraHaan** [comment]
I think it is because of the fact that the ``{`` is on the source line, after I had something like that happen for a field initializer I tried putting the open brace on a new line and it aligned it properly afterwards.

**Comment by Blinnikov** [comment]
Hi!
When it's going to be released?
